src/hooks/useStatsig.tsx

```diff
@@ -28,12 +33,14 @@ export const useStatsigGateValue = (gate: StatSigFlags) => {
   return checkGate(gate);
 };
 
-export const useStatsigDynamicConfigValue = (
-  configName: StatsigDynamicConfigs,
-  keyOverride: string = 'value'
-) => {
+export const useAllStatsigDynamicConfigValues = () => {
   const { getDynamicConfig } = useStatsigClient();
-  return getDynamicConfig(configName)?.get(keyOverride);
+  const allDynamicConfigValues = useMemo(() => {
+    return Object.values(StatsigDynamicConfigs).reduce((acc, gate) => {
+      return { ...acc, [gate]: getDynamicConfig(gate).get('value') };
```

Comment:
got rid of keyoverride but it's not used anywhere rn so I figure we can add it back if/when we need it

Reply:
is there a risk that getDynamicConfig stays the same but the result of calling getDynamicConfig() changes so this memoized object gets stale? e.g. if on startup getDynamicConfig() returns undefined but after some initialization it has a value?

Reply:
yeah :'( gonna tag @yogurtandjam just as an FYI when he gets back but no action needed for now!